apps/web/app/api/generate/route.ts

```diff
@@ -0,0 +1,51 @@
+import { OpenAIStream, StreamingTextResponse } from "ai";
+import { getOpenAI } from "@/utils/openai";
+import { auth } from "@/app/api/auth/[...nextauth]/auth";
+import { NextResponse } from "next/server";
+import { DEFAULT_AI_MODEL } from "@/utils/config";
+import { withError } from "@/utils/middleware";
+
+export const POST = withError(async (request: Request): Promise<Response> => {
+  const session = await auth();
+  if (!session?.user.email)
+    return NextResponse.json({ error: "Not authenticated" });
+
+  const user = await prisma.user.findUniqueOrThrow({
```

Comment:
Missing

```ts
import prisma from "@/utils/prisma";
```

Surprised this didn't error for you.

Reply:
looks like i forgot to import. how the code works without imports? :)

Does these errors catched by pre-commit hooks?

Reply:
yes, i think the deploy failed as well (but i had to first allow the deploy to happen on vercel).
we could add github action checks too to catch them sooner